src/meatie/option/cache_option.py

```diff
@@ -57,18 +58,10 @@ def get_key(request: Request) -> str:
     key = request.path
     if request.params:
         key += "?" + urllib.parse.urlencode(request.params)
+    # TODO (HomerusJa): Consider adding the request body's hash to the key
```

Comment:
I am not conviced body has should be used. Overall, caching should only be done for idempotent actions, in other words GET methods in practice. Sending body in a GET request although possible in theory has no semantic meaning to the server.